apps/server/src/controllers/userControllers/authUsers.ts

```diff
@@ -360,3 +362,68 @@ export const resetPassword = async (req: Request, res: Response) => {
     })
   }
 }
+
+export const changePassword = async (req: RequestExt, res: Response) => {
+  const passwordRegex = passwordUtils.regex
+
+  try {
+    const { _userId: userId, ...rest } = req.body
+
+    const requestData = changePasswordSchema.strict().safeParse(rest)
```

Comment:
You have to handle the _user  in the rest object before validating 
or validate using safeParse alone


const requestData = changePasswordSchema.safeParse(rest)